seismic_hazard_analysis/utils.py

```diff
@@ -218,3 +228,152 @@ def get_im_levels(im: str, n_values: int = 100):
         start=np.log(start), stop=np.log(end), num=n_values, base=np.e
     )
     return im_values
+
+
+
+def rp_to_prob(rp: float, t: float = 1.0):
+    """
+    Converts return period to exceedance probability
+    Based on Poisson distribution
+
+    Parameters
+    ----------
+    rp: float
+        Return period
+    t: float
+        Time period of interest
+
+    Returns
+    -------
+    Exceedance probability
+    """
+    return 1 - np.exp(-t / rp)
+
+
+def prob_to_rp(prob: float, t: float = 1.0):
+    """
+    Converts probability of exceedance to return period
+    Based on Poisson distribution
+
+    Parameters
+    ----------
+    prob: float
+        Exceedance probability
+    t: float
+        Time period of interest
+
+    Returns
+    -------
+    Return Period
+    """
+    return -t / np.log(1 - prob)
+
+
+def exceedance_to_im(
+    exceedances: np.ndarray, im_values: np.ndarray, hazard_values: np.ndarray
+):
+    """
+    Converts the given exceedance rate to an IM value, based on the
+    provided im and hazard values
```

Comment:
Small formatting suggestion
```suggestion
    provided im and hazard values.
    
```